Wikipedia/Code/NSURL+WMFLinkParsing.m

```diff
@@ -77,7 +77,7 @@ - (NSString*)wmf_domain {
 
 - (NSString*)wmf_language {
     NSRange dotRange = [self.host rangeOfString:@"."];
-    if (dotRange.length == 1 && dotRange.location < 4 && dotRange.location > 1) {
+    if (dotRange.location != NSNotFound) {
```

Comment:
@joewalsh Good catch. `test` wiki would be another one. See the list here: https://en.wikipedia.org/wiki/List_of_Wikipedias